src/views/Player.vue

```diff
@@ -258,11 +297,37 @@ export default {
 
         // initializing the retention array with zeros
         this.retention = Array(this.player.duration).fill(0);
+        this.submitted = Array(this.ivideo_questions.length).fill(0);
+        this.skipped = Array(this.ivideo_questions.length).fill(0);
+        this.revised = Array(this.ivideo_questions.length).fill(0);
+
+        const play_buttons = document.querySelectorAll("[data-plyr='play']")
+        play_buttons[0].addEventListener(
+          'click', this.listenToPlayButtons, false
+        );
+
+        play_buttons[1].addEventListener(
+          'click', this.listenToPlayButtons, false
+        );
+
```

Comment:
2 buttons?

Reply:
yes. A big one in the middle of the screen and a smaller one in the bottom left corner

Reply:
Add this information as a comment in the code